Sources/UtilityBeltUITesting/Enumerations/LaunchArgument.swift

```diff
@@ -0,0 +1,8 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+/// An enumeration of launch arguments used when launching an XCUIApplication.
+public enum LaunchArgument: String {
+    case xcuiTestCase
+}
```

Comment:
How would we handle launch arguments that are specific to an individual app since we wouldn't be able to extend the enum to add new cases?